src/UI/Text.elm

```diff
@@ -1,110 +1,203 @@
 module UI.Text exposing
     ( Text
-    , TextColor
-    , body1
-    , body2
-    , caption
-    , combination
-    , heading1
-    , heading2
-    , heading3
-    , heading4
-    , heading5
-    , heading6
-    , multiline
-    , overline
-    , renderElement
-    , setEllipsis
-    , subtitle1
-    , subtitle2
+    , body1, body2
+    , heading1, heading2, heading3, heading4, heading5, heading6
+    , subtitle1, subtitle2
+    , caption, overline
+    , multiline, combination
     , withColor
+    , setEllipsis
+    , renderElement
     )
 
+{-| `UI.Text` is a component to specify how text to display text. It applies font size, weight, letter-spacing, and color.
```

Comment:
I would enforce in the description that we should not be using Element.text directly in paack's project 🤔